tools/Install-XHarness.ps1

```diff
@@ -0,0 +1,31 @@
+### This script is a quick way to install XHarness in a local folder.
+### .NET SDK is installed too, everything in a local folder which can be deleted afterwards.
+###
+### Use the following command to run this script from anywhere:
+### iex ((New-Object System.Net.WebClient).DownloadString('https://aka.ms/get-xharness-ps1'))
+
+param (
+    [Parameter(Mandatory = $false)]
+    [string]
+    $Version = "*"
+)
+
+$ErrorActionPreference = "Stop"
+
+$xharness_version = "1.0.0-prerelease.$Version"
+
+# Install .NET
+Write-Host "Getting dotnet-install.ps1.." -ForegroundColor Cyan
+Invoke-WebRequest -Uri "https://dot.net/v1/dotnet-install.ps1" -OutFile "dotnet-install.ps1"
+
+Write-Host "Installing .NET SDK locally to " -NoNewline -ForegroundColor Cyan
+Write-Host ".dotnet" -ForegroundColor Yellow
+Invoke-Expression -Command "& ./dotnet-install.ps1 -InstallDir ./.dotnet -Version 6.0.100"
```

Comment:
You don't need `Invoke-Expression` here. You can just directly call what you have in `-Command` or even skip `&` if there are no spaces in the filename.
```suggestion
./dotnet-install.ps1 -InstallDir ./.dotnet -Version 6.0.100
```

Reply:
Of course.. This line came from the copilot btw 😅

Reply:
I was too happy it gave me the full line that I forgot to read it